test/test_collections.rb

```diff
@@ -173,6 +173,48 @@ class TestCollections < JekyllUnitTest
     end
   end
 
+  context "with a collection with metadata to sort items by attribute" do
+    setup do
+      @site = fixture_site({
+        "collections" => {
+          "methods"   => {
+            "output" => true,
+          },
+          "tutorials" => {
+            "output"  => true,
+            "sort_by" => "lesson",
+          },
+        },
+      })
+      @site.process
+      @tutorials_collection = @site.collections["tutorials"]
+
+      @actual_array = @tutorials_collection.docs.map(&:relative_path)
```

Comment:
Let's call this `@collection_doc_relative_paths` or something. `@actual_array` is a description of the data structure, not the content.